### PR TITLE
For non-istio apps make correct the namespace/validation spacing

### PR DESCRIPTION
**Pull request by mtho11**
** Describe the change **
For regular k8s apps, meaning non-istio (without sidecars) apps, correct the namespace spacing and validations to look proper.

fixes #https://github.com/kiali/kiali/issues/2217

** Screenshot **

Before:
Kiali_Console_-_Firefox_Developer_Edition

After:
Kiali_Console

### CODE REVIEW

File: src/pages/Graph/SummaryPanelGraph.tsx [hunk 1]
```diff
@@ -275,7 +275,7 @@ export default class SummaryPanelGraph extends React.Component<SummaryPanelPropT
               NS
             </Badge>
           </Tooltip>
-          {ns}
+          {ns}{' '}
```
Comment (jshaughn) [inline src/pages/Graph/SummaryPanelGraph.tsx]: Doesn't this add space in all cases?  This is minor and I guess ir doesn't really matter but this way there still may be inconsistent space between the namespace name and the validation status. No big deal, can leave it.

Comment (mtho11) [reply src/pages/Graph/SummaryPanelGraph.tsx]: Yes, it does but I think it is irrelevant given it keeps the codes base simple and the human eye can barely discriminate the extra 5 pixels:

Kiali_Console

Comment (mtho11) [reply src/pages/Graph/SummaryPanelGraph.tsx]: I'm totally willing to remedy this if anyone feels it warrants the extra 'if' clause.

Comment (mtho11) [reply src/pages/Graph/SummaryPanelGraph.tsx]: Here is an image without the extra 5px spacing:
Kiali_Console

Hardly discernible to the human eye.

But again, I'm open to whatever ppl think.

Comment (xeviknal) [reply src/pages/Graph/SummaryPanelGraph.tsx]: It make sense. And it keeps the code cleaner, I guess.

**Comment by pbajjuri20** [comment]
Tested and passed on Kiali UI1.14.0 (d4faab6b59ca7c710b11ecd37276be6dcfac4ebe)
Kiali Server v1.14.0-SNAPSHOT (069101f97dbe96bfeb0bc9172702076c5ad09898)

**Pull request merged**